.gitlab-ci.yml

```diff
@@ -111,7 +113,28 @@ test:debug:
     reports:
       junit: "**/build/test-results/testDebugUnitTest/*.xml"
 
-test:release:
+test:debug:
```

Comment:
I can only see where you disable the `ITR` tests here but not where you enable them. Does it mean that before they were using the ITR in the `dd-java-agent` by default ?